components/Card.js

```diff
@@ -6,8 +6,12 @@ import MostReqTasks from './MostReqTasks'
 
 export default function Card(props) {
   const [isOpen, setIsOpen] = useState(false)
+
+  var tasks = props.taskGroups
   const mostReq = props.taskGroups[0]
-  const tasks = props.taskGroups.slice(1, props.taskGroups.length)
+  if (props.mostReq) {
+    tasks = props.taskGroups.slice(1, props.taskGroups.length)
```

Comment:
This is fairly confusing. I proposed an idea to @shewood that I think would work well

Remove this code and then below we can do
<MostReqTasks taskList={props.taskGroups} mostReq={true} />